common_upstream_dependencies.repos

```diff
@@ -12,3 +12,8 @@ repositories:
     type: git
     url: https://github.com/dawonn/vectornav
     version: 1ecaea8966eb04507d7a80b86a21860e4941928f # Latest known working on ros2 branch
+
+  zed2_ros_wrapper:
+    type: git
+    url: https://github.com/stereolabs/zed-ros2-wrapper.git
+    version: master
```

Comment:
I'd like specifying a specific commit here instead of `master` such that our repo doesn't randomly break in the case that they change something on the `stereolabs/zed-ros2-wrapper` side.